ui/src/core/components/Modal/Wizard/index.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useRef, useState, useEffect } from 'react';
+import indexOf from 'lodash/indexOf';
+import map from 'lodash/map';
+import Text from 'core/components/Text';
+import { WizardItems } from './constants';
+import Styled from './styled';
+
+interface Item {
+  icon: string;
+  title: string;
+  name: string;
+  size: string;
+  backgroundColor: string;
+  subtitle: string;
+}
+
+const Wizard = () => {
+  const modalRef = useRef<HTMLDivElement>();
+  const [itemSelect, setItemSelect] = useState<Item>(WizardItems[0]);
+  const [indexOfItemSelect, setIndexOfItemSelect] = useState(0);
+  const [localStorageValue, setLocalStorageValue] = useState("");
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    setIndexOfItemSelect(indexOf(WizardItems, itemSelect));
+  }, [itemSelect]);
+
+  useEffect(() => {
+    setItemSelect(WizardItems[indexOfItemSelect]);
+  }, [indexOfItemSelect]);
+
+  useEffect(() => {
+    setLocalStorageValue(localStorage.getItem("wizard"));
+  }, []);
+
+  useEffect(() => {
+    if (!localStorageValue) {
+      setIsOpen(true);
+    } else if (localStorageValue === "true") {
+      setIsOpen(false);
+    }
+  }, [localStorageValue]);
+
+  const isFinalStep = () =>
+    itemSelect.name === 'metrics-provider' ? true : false;
```

Comment:
```suggestion
    itemSelect.name === 'metrics-provider';
```
You could just return the condition